source/_components/sensor.fronius.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: "Fronius "
+description: "Instructions on how to connect your Fronius Inverter to Home Assistant."
+date: 2018-01-05 11:43:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Energy
+logo: fronius.png
+ha_iot_class: "Local Polling"
+ha_release: 0.60
+---
+
+
+The `fronius` sensor will poll a [Fronius](http://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors (or attributes of sensors) in Home Assistant.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
```

Comment:
We like to keep the initial example as minimal as possible.
Only include 1 example, with only the required configuration variables (leave optional ones out).

It is ok to add more examples later on.